app/services/usps_daily_test_sender.rb

```diff
@@ -0,0 +1,47 @@
+# Mails a letter to the designated receiver
+class UspsDailyTestSender
+  def run
+    if valid_designated_receiver_pii?
+      UspsConfirmationMaker.new(
+        pii: designated_receiver_pii,
+        issuer: nil, # TODO: login.gov internal issuer?
```

Comment:
@stevegsa do you know of a good issuer we could add here to not mess up billing? or is `nil` OK?

Reply:
I think internal issuer would be better

Reply:
I just checked our list of issuers we don't have a good issuer in our prod environment (only our lower sandbox ones), so based on [Steve's Slack comment](https://gsa-tts.slack.com/archives/C5E7EJWF7/p1609824563055400?thread_ts=1609790670.054700&cid=C5E7EJWF7) going to leave it blank for now

Reply:
blank is good